src/main/java/com/flippingutilities/model/AccountData.java

```diff
@@ -77,6 +77,7 @@ public void prepareForUse(FlippingPlugin plugin)
 			item.setOfferMadeBy();
 			item.setTotalGELimit(geLimit);
 			item.syncState();
+			item.setOfferIds();
```

Comment:
preparation for the combination flip stuff, you can ignore these, just setting unique ids on all offers